library/ajax/weno_pharmacy_search.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * weno pharmacy search.
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Kofi Appiah <kkappiah@medsov.com>
+ * @copyright Copyright (c) 2023 Kofi Appiah <kkappiah@medsov.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+ require_once(dirname(__FILE__) . "/../../interface/globals.php");
+
+if (isset($_GET['searchFor']) && $_GET['searchFor'] == 'weno_city') {
+    $return_arr = array();
+    $term    = filter_input(INPUT_GET, "term");
+
+    $sql = "SELECT city, id FROM weno_pharmacy WHERE city LIKE ? LIMIT 10";
+    $val = array('%' . $term . '%');
+    $res = sqlStatement($sql, $val, $val);
```

Comment:
above will break since only 1 parameter in your sql query (ie. only need one $val parameter)